core/src/syn/parser/stmt/define.rs

```diff
@@ -31,7 +31,9 @@ use crate::{
 impl Parser<'_> {
 	pub async fn parse_define_stmt(&mut self, ctx: &mut Stk) -> ParseResult<DefineStatement> {
 		match self.next().kind {
-			t!("NAMESPACE") => self.parse_define_namespace().map(DefineStatement::Namespace),
+			t!("NAMESPACE") | t!("ns") => {
```

Comment:
Shouldn't this be `NS` to follow the naming convention used for the other tokens?

Reply:
The `ns` here isn't a keyword, all the keywords have uppercase names, but `ns` her is the `DurationSuffix(DurationSuffix::Nano)` token. The `ns` part in a duration like `1s30ns`. This token can only be lowercase which is why the `t!("ns")` is also lowercase.